WTWSMS/localisation/WtWSMS_titles.csv

```diff
@@ -273,8 +275,8 @@ k_rugiland;Rugiland;;;;;;;;;;;x
 k_rugiland_adj;Rugii;Ruge;;;;;;;;;;x
 k_herul;Herul;;;;;;;;;;;x
 k_herul_adj;Heruli;;;;;;;;;;;x
-k_ostrogoths;Ostrogoths;Ostrogoths;;;;;;;;;;x
-k_ostrogoths_adj;Ostrogothic;Ostrogoth;;;;;;;;;;x
+d_ostrogoths;Ostrogoths;Ostrogoths;;;;;;;;;;x
+d_ostrogoths_adj;Ostrogothic;Ostrogoth;;;;;;;;;;x
```

Comment:
Good news! I just checked my localization files, and BOTH titles are present, duchy and kingdom.